omod/src/main/resources/sqldiff.xml

```diff
@@ -431,11 +431,9 @@
 			`billing_patient_service_bill_item`
 		</description>
 		<sql>
-<!-- 			ALTER TABLE `billing_patient_service_bill` ADD `actual_amount`
-			decimal(19,2) DEFAULT NULL;  -->
-			ALTER TABLE
```

Comment:
@bkirui why have you removed the script that alters `billing_patient_service_bill_item` (note it is different from `billing_patient_service_bill`)